node/src/Pos/Wallet/Web/Pending/Functions.hs

```diff
@@ -0,0 +1,176 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+
```

Comment:
We explictly ignored this piece of work, available originally [here](https://github.com/input-output-hk/cardano-sl/pull/2213/commits/b13933bf5dc608e0e366e13b6f318f3fa898d567), and decided not to port it.